packages/react-components/react-theme/package.json

```diff
@@ -30,7 +29,8 @@
     "@fluentui/scripts": "^1.0.0"
   },
   "dependencies": {
-    "tslib": "^2.1.0"
+    "tslib": "^2.1.0",
```

Comment:
It's unrelated, but can you please cleanup `peerDependencies`? `react` & `react-dom` are not used there at all.